dev/watch.js

```diff
@@ -96,6 +97,9 @@ chokidar.watch(`${sassDir}/**/*.scss`).on('change', changedFile => {
                 filesToCompile.map(file => file.replace('scss', 'css'))
             );
         })
+        .then(() => {
+            copyFiles.task();
```

Comment:
would `useHashedBundles` cover that? you'd need to reload manually i think, but with that set to true, it would act like prod?